packages/design-tokens/src/tokens/functional/components/bento/base.json

```
@@ -0,0 +1,43 @@
+{
+  "brand": {
+    "Bento": {
+      "gap": {
+        "large": {
+          "value": "var(--base-size-32)"
+        },
+        "medium": {
+          "value": "var(--base-size-24)"
+        },
+        "small": {
+          "value": "var(--base-size-20)"
+        }
+      },
+      "Item": {
+        "borderRadius": {
+          "small": {
+            "value": "16px"
+          },
+          "medium": {
+            "value": "20px"
+          },
+          "large": {
+            "value": "24px"
+          }
```

Do these need to be px instead of using our rem / design token values?

```suggestion
          "small": {
            "value": "var(--base-size-16)"
          },
          "medium": {
            "value": "var(--base-size-20)"
          },
          "large": {
            "value": "var(--base-size-24)"
          }
```